demisto_sdk/scripts/init_validation_script.py

```diff
@@ -568,23 +617,57 @@ def generate_file_info(self):
             self.file_name = pascal_to_snake(self.class_declaration[6:-39])
         if not self.file_name.startswith(self.error_code):
             self.file_name = f"{self.error_code}_{self.file_name}"
-        if not self.file_name.endswith(".py"):
-            self.file_name = f"{self.file_name}.py"
+        if self.file_name.endswith(".py"):
+            self.file_name = f"{self.file_name[0:-3]}"
```

Comment:
can you explain this fix?

Reply:
we need the file name without the suffix for use in other places

Reply:
I mean explain the change. Before your changes the code would make sure all files ends with `.py`:
```python
if not self.file_name.endswith(".py"):
            self.file_name = f"{self.file_name}.py"
```
You code is doing the opposite

Reply:
True, I ensure that the name does not include the suffix, and then when creating the file I always add it.
And I changed it, because in the other files I need to add "all-files"/"list-files" to the file name